src/client/datascience/variablesView/variableView.ts

```diff
@@ -182,6 +184,9 @@ export class VariableView extends WebviewViewHost<IVariableViewPanelMapping> imp
             const response = await this.variables.getVariables(args, this.notebookWatcher.activeNotebook);
 
             this.postMessage(InteractiveWindowMessages.GetVariablesResponse, response).ignoreErrors();
+            sendTelemetryEvent(Telemetry.VariableExplorerVariableCount, undefined, {
```

Comment:
This is same event in the same spot as with the old webviews, lets us see how many variables we are fetching.